src/components/visualizations/chart/legend/StaticLegend.tsx

```diff
@@ -77,13 +98,8 @@ export default class StaticLegend extends React.PureComponent<any, any> {
 
         const pagedSeries = series.slice(start, end);
 
-        // For Geo Pushpin with position left/right
-        // we set the height of series to number of actual displayed items
-        // so that, size legend will be visible
-        const isPushpinAndLeftRightPosition =
-            isPushpin(chartType) && (position === LEFT || position === RIGHT);
         const seriesHeight =
-            (isPushpinAndLeftRightPosition ? Math.min(visibleItemsCount, seriesCount) : visibleItemsCount) *
+            (shouldFillAvailableSpace ? visibleItemsCount : Math.min(visibleItemsCount, seriesCount)) *
             ITEM_HEIGHT;
```

Comment:
Please assign this `Math.min(visibleItemsCount, seriesCount)) * ITEM_HEIGHT` to meaningful name const

Reply:
updated :)